.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -8,28 +8,18 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+<!-- whole examples/explanation text here -->

```

Comment:
What I meant was that we should *hide* the example part and `Steps to reproduce the behavior:` with a comment, not write that dummy string above.